### PR TITLE
avatar: convert stories to CSF

### PR DESCRIPTION
**Pull request by danethurber**

### CODE REVIEW

File: packages/avatar/src/react/__stories__/index.story.tsx [hunk 1]
```diff
@@ -1,107 +1,120 @@
...
+SizeSmall.args = { ...defaultArgs, size: Avatar.sizes.small }
+
+export const SizeMedium = Template.bind({})
+SizeMedium.args = { ...defaultArgs, size: Avatar.sizes.medium }
+
+export const SizeLarge = Template.bind({})
+SizeLarge.args = { ...defaultArgs, size: Avatar.sizes.large }
+
+export const SizeXLarge = Template.bind({})
+SizeXLarge.args = { ...defaultArgs, size: Avatar.sizes.xLarge }
```
Comment (jaketrent) [inline packages/avatar/src/react/__stories__/index.story.tsx:114]: Could convert size stories to a single story.

File: packages/avatar/src/react/__stories__/index.story.tsx [hunk 2]
```diff
@@ -1,107 +1,120 @@
...
+NameUndefined.args = { ...defaultArgs, name: undefined }
+
+export const NameSingleChar = Template.bind({})
+NameSingleChar.args = { ...defaultArgs, name: 'x' }
+
+export const NameSpecialChar = Template.bind({})
+NameSpecialChar.args = { ...defaultArgs, name: '% Special Char' }
+
+export const VariousNames: Story = () => {
+  const names = useMemo(
```
Comment (jaketrent) [inline packages/avatar/src/react/__stories__/index.story.tsx:37]: Why is useMemo required static data like this? And why required in a story?

**Review by jaketrent — APPROVED** [review]
Story format looks great. 

As you likely agree, the effort to import stories into specs seems bespoke and will need to be remembered per component. This will require some discipline from all who do these upgrades.

Not sure if this is a regression: 
The "Image Error" story shows the alt text in large type (ie, "Jake Trent") in the too-small space before the image fails and the initials are finally shown.

File: packages/avatar/src/react/__stories__/index.story.tsx (hunk 2 above)
Comment (danethurber) [reply packages/avatar/src/react/__stories__/index.story.tsx:37]: uh...no reason. just me doing weird things. i'll fix

**Pull request merged**